src/Header/DownloadStream.h

```diff
@@ -25,7 +25,9 @@ class DownloadStream {
   // filename include file path
   std::string filename;
   // default chunk size
-  unsigned long chuck_size;
+  unsigned long chunk_size;
+  // True indicate that it has the read access of the file, otherwise.
+  bool reading = false;
```

Comment:
do we initialize variable in class declaration?

Reply:
Not sure the google style mention about this, it has been fixed, after all.